docs/Home.md

```diff
@@ -1,3 +1,50 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Ex: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria
+
+How do we decide if a feature is **essential**? 👍
+
+- Our team requires it to maintain code quality
+- Our team requires it to maintain a good developer experience
+- Our team requires it to increase the productivity of the team
+- It is always used or configured in the projects we've worked on
+- It is recommended to use or configure based on Google's standards
+
+How do we decide if a feature is **optional**? 🚩
+
+- It is regularly used or configured in projects we've worked on
+- It is regularly used or configured as an alternative to the default choice
+
+How do we decide if a feature is **rejected**? 👎
+
+- It is barely used or configured in projects we've worked on
+- It is barely used or configured as an alternative to the default choice
+
+Let's have a look at some examples 🔎
+
+**Essential**:
+- [Jacoco](https://github.com/jacoco/jacoco): Code coverage is important to maintain code quality.
+- [Hilt](https://developer.android.com/training/dependency-injection/hilt-android): Dependency injection provides a good developer experience.
+- [Firebase App Distribution](https://firebase.google.com/docs/app-distribution): Continuous delivery increases the productivity of the team.
+- [Glide](https://github.com/bumptech/glide): Image loading is always used in all the projects we've worked on.
```

Comment:
@Tuubz I think guide it's an `optional` we can add it later when we have to implement feature with `image/video`  🤔 

But I think would it be good if we add `RxJava and Coroutine` in `essential` as we know we must to choose one of both before create project and what about `navigation` component, Is it essential too like `Compose` or `Hilt` What do you think 🙏

Reply:
@Wadeewee Thank you for your feedback 🚀 

> I think guide it's an optional we can add it later when we have to implement feature with image/video

In my opinion, I think `Glide` is `essential`. Because it is always used in all our previous projects. I think it's very unlikely that a future project will not have any image loading. Please correct me if I'm wrong 🙇 

> But I think would it be good if we add RxJava and Coroutine in essential as we know we must to choose one of both before create project

This is a bit of a special case, RxJava should actually be re-factored to be `optional`. While `Coroutine` should be `essential` and set as our default, as it's recommended by Google. What do you think?

> what about navigation component, Is it essential too like Compose or Hilt

Yes, the navigation component is also essential. This is because it is recommended by Google's standards 🙏

Reply:
>This is a bit of a special case, RxJava should actually be re-factored to be optional. While Coroutine + LiveData (or Flow) should be essential and set as our default, as it's recommended by Google. What do you think?
>
>Yes, navigation component is also essential. This is because it is recommended by Google's standards 

@Tuubz I agreed with this so let add `Coroutine + Flow` and  `navigation ` in `essential` and  add  `RxJava`  in `optional` but for `LiveData` I think it old and we don't use it in `CoroutineTemplate` 🙏  and one more thing, Would it be great if we reference the example by the dependencies that we're using in our both template 🙇‍♀️

Reply:
> I agreed with this so let add Coroutine + Flow and navigation in essential and add RxJava in optional but for LiveData I think it old and we don't use it in CoroutineTemplate 🙏

For essential features, there are five criteria, so my idea is to give _one_ example for each criterion:

- I could replace the `Compose` example with `Navigation Component`, would that be better?
- Since `Coroutine` & `RxJava` are special cases and will be refactored in the future, I think it's better to avoid these examples (for now), to avoid confusion. What do you think? 🙏 

> Would it be great if we reference the example by the dependencies that we're using in our both template

Ex: Are you suggesting we should avoid using `Jacoco` as an example, because it's currently only used as a dependency for `RxJava`, but not for `Coroutine`? 🙇

Reply:
> I could replace the Compose example with Navigation Component, would that be better?
Since Coroutine & RxJava are special cases and should be refactored in the future, I think it's better to avoid these examples (for now), to avoid confusion. What do you think? 🙏

@Tuubz Could you please explain more bit about `Since Coroutine & RxJava are special cases and should be refactored in the future` Didn't get it much 🙏 

> Ex: Are you suggesting we should avoid using Jacoco as an example, because it's currently only used in RxJava, but not for Coroutine? 🙇 I guess we could replace this example with Detekt instead 🚀
>

@Tuubz Sound good to me from now we can consider to add `Jacoco` later 🚀

Reply:
Ah, I see @Tuubz thanks for explanation 🙇‍♀️